lib/game.rb

```diff
@@ -2,6 +2,8 @@
 
 # Game
 class Game
+  @@games = []
```

Comment:
I think we should review this class variable, to insure we all understand how it's working